### PR TITLE
semi_supervised depth-pred

### PR DESCRIPTION
**Pull request by faezehamjadi**

### CODE REVIEW

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: review
+title: "Semi_Supervised Deep Learning for Monocular Depth Map Prediction"
+tags: semi_supervise for predict depth
```
Comment (Dref360) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: use single words. Like `semi-supervised depth-estimation`

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+author: Faezeh Amjadi
+cite:
+    authors: "Yevhen Kuznietsov, jorg stuckler, Bastin Leibe"
+    title:   "Semi_Supervised Deep Learning for Monocular Depth Map Prediction"
+    venue:   "CVPR 2017"
+pdf:   "https://arxiv.org/pdf/1702.02706.pdf"
+---
+
+The author proposes a depth map prediction in a
```
Comment (Dref360) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: authors* propose*

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+layout: review
+title: "Semi_Supervised Deep Learning for Monocular Depth Map Prediction"
+tags: semi_supervise for predict depth
+author: Faezeh Amjadi
+cite:
+    authors: "Yevhen Kuznietsov, jorg stuckler, Bastin Leibe"
```
Comment (Dref360) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: Jorg Stuckler*

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    venue:   "CVPR 2017"
+pdf:   "https://arxiv.org/pdf/1702.02706.pdf"
+---
+
+The author proposes a depth map prediction in a
+semi-supervised way for a monocular image. The network is a CNN (a deep residual network architecture in an encoder-decoder scheme) and output of the network is a disparity prediction (or inverse of depth).
+
+The network is trained by unsupervised and supervised depth cues. The supervised part compares the output of network with the ground truth.
+
+The unsupervised part learns depth prediction, directly from the stereo images that the left image was feed to the network, so with the left image and disparity (output of the network), each pixel from the left image can be mapped to the right image and inverse. Thus, the author defines loss by direct image alignment error in both directions in this part. [ unsupervised because the network doesnt use ground truth to calculate the loss].
```
Comment (Dref360) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: fed*

Comment (Dref360) [reply deep-learning/_posts/2017-09-06-semi_predDepth.md]: Authors* define*

Comment (Dref360) [reply deep-learning/_posts/2017-09-06-semi_predDepth.md]: doesn't*

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+
+An **important** note is that the author use BerHu_norm indsead of L2_norm in supervised loss that performs better on test set and reduce the noise.
```
Comment (Dref360) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: instead*

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+layout: review
+title: "Semi_Supervis for  Depth Prediction"
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: The title of the paper is
>Semi-Supervised Deep Learning for Monocular Depth Map Prediction

I would suggest to shorten this to: **Semi-Supervised Monocular Depth Map Prediction**

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+tags: semi_supervise for predict depth
+author: Faezeh Amjadi
+cite:
+    authors: "Yevhen Kuznietsov, Jorg Stuckler, Bastin Leibe"
+    title:   "Semi_Supervised Deep Learning for Monocular Depth Map Prediction"
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: Semi-Supervised

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    venue:   "CVPR 2017"
+pdf:   "https://arxiv.org/pdf/1702.02706.pdf"
+---
+
+The authors propose a depth map prediction in a
+semi-supervised way for a monocular image. The network is a CNN (a deep residual network architecture in an encoder-decoder scheme) and output of the network is a disparity prediction (or inverse of depth).
+
+The network is trained by unsupervised and supervised depth cues. The supervised part compares the output of network with the ground truth.
+
+The unsupervised part learns depth prediction, directly from the stereo images that the left image was fed to the network, so with the left image and disparity (output of the network), each pixel from the left image can be mapped to the right image and inverse. Thus, the authors define loss by direct image alignment error in both directions in this part. [ unsupervised because the network does not use ground truth to calculate the loss].
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: Please drop the `[ ]` and add something like this that the beginning: "We call this unsupervised because the network..."

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+
+An **important** note is that the author use BerHu_norm instead of L2_norm in supervised loss that performs better on test set and reduce the noise.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: Please
* remove the `_`
* add a link to the BerHu paper like this: `[BerHu norm](http://...)`

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+
+An **important** note is that the author use BerHu_norm instead of L2_norm in supervised loss that performs better on test set and reduce the noise. 
+
+<div align="middle">
+  <img src="/deep-learning/images/pred_depth/1.png" width="400">
+</div>
+
+
+## Experiments and results
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: Can you give an idea of quantitative results? Maybe add one sentence or add a figure

File: deep-learning/_posts/2017-09-06-semi_predDepth.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+
+<div align="middle">
+  <img src="/deep-learning/images/pred_depth/2.png" width="600">
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-06-semi_predDepth.md]: I think this figure does not belong in the _Experiments_ section.
